routes/web.php

```diff
@@ -14,3 +14,5 @@
 Route::get('/', function () {
     return view('welcome');
 });
+Auth::routes();
+Route::get('/home', 'HomeController@index')->name('home');
```

Comment:
/home is not implemented yet, so you can remove this for now.

Reply:
Fixes issue(Delete)

Reply:
Fixes issue(Delete)